api/views.py

```diff
@@ -86,6 +94,26 @@ def post(self, request, pk):
     return Response(LessonStudentSerializer(new_lessonstudent).data)
 
 
+class StudentAverage(APIView):
+  parser_classes = [JSONParser]
+
+  def get(self, request, pk):
+    average_score = LessonStudent.student_average_score(pk)['score__avg']
+    student_score = StudentScore(student_id=pk, average_score=average_score)
+    
+    return Response(StudentScoreSerializer(student_score).data)
+
+
+class LessonAverage(APIView):
+  parser_classes = [JSONParser]
+
+  def get(self, request, pk):
+    average_score = LessonStudent.lesson_average_score(pk)['score__avg']
+    lesson_score = LessonScore(lesson_id=pk, average_score=average_score)
+    
+    return Response(LessonScoreSerializer(lesson_score).data)
+
+
```

Comment:
Great job here, it was straightforward and simple. Tracking back through the serializer and model method helped me put this all together.